freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_backup

```diff
@@ -158,8 +176,12 @@ exlock_now || error_exit "A backup seems to be currently running. Lock file is a
 #set -x
 
 # Perform a backup
-/sbin/ipa-backup $BACKUP_OPTIONS >> $LOGFILE 2>&1 || error_exit "ipa-backup failed! Aborting!"
-BACKUPDIR=$(basename $(ls -td ${config[backup_path]}/ipa-* | head -1))
+
+# shellcheck disable=SC2086
+/sbin/ipa-backup ${BACKUP_OPTIONS} >> "${LOGFILE}" 2>&1 || error_exit "ipa-backup failed! Aborting!"
+
+# shellcheck disable=SC2012
+BACKUPDIR=$(basename "$(ls -td "${config[backup_path]}""/ipa-*" | head -1)")
```

Comment:
Is there a reason the double quotes are back to back?
Wouldn't `BACKUPDIR=$(basename "$(ls -td "${config[backup_path]}/ipa-*" | head -1)")` be simpler?

Reply:
fixed